rapidfuzz/details/matching_blocks.hpp

```diff
@@ -56,36 +56,31 @@ class SequenceMatcher {
         }
     }
 
-    match_t find_longest_match(size_t a_low, size_t a_high, size_t b_low, size_t b_high)
+    match_t find_longest_match(std::size_t a_low, std::size_t a_high, std::size_t b_low, std::size_t b_high)
     {
-        size_t best_i = a_low;
-        size_t best_j = b_low;
-        size_t best_size = 0;
+		std::size_t best_i = a_low;
```

Comment:
I am fine with the use of std::size_t, but the alignment should stay the same

Reply:
Hmm. That is peculiar. On my end it looks perfectly aligned. Perhaps the misalignment is due to my vim settings.

Reply:
![1631495899](https://user-images.githubusercontent.com/24488175/133010711-ca60cb6f-7434-411e-870c-aafdf4555e47.png)
![1631495909](https://user-images.githubusercontent.com/24488175/133010713-0fc38fd3-af3e-4d42-9b92-3dbd7e27ed7b.png)

VSCode & Vim both show identical indentation. Perhaps its GitHub misinterpreting the tab/space conversion?
I have had this happen a few times before as well, never figured out the root cause, but i suspect its the tab to space conversion.

Reply:
Yes, GitHub displays tabs as 8 white spaces by default. Could you replace the tabs with 4 white spaces for consistency with the rest of the code base?

Reply:
I have tried using spaces to indent. See if the latest commit looks fine.

Reply:
Yes, it looks good now :+1: